vevestaX/vevesta.py

```diff
@@ -106,6 +106,29 @@ def endModelling(self):
     # create alias of method modellingStart and modellingEnd
     start = startModelling
     end = endModelling
+    
+    
+    # function to get arguments of a function
+    def parameters(self, func):
+        def wrapper(*args, **kwargs):
+            # to get parameters of function that are passed
+            functionParameters = inspect.signature(func).bind(*args, **kwargs).arguments
+            functionParameters = dict(functionParameters)
+            # to get values that are not passed and defautlt values
+            defaultParameters = inspect.signature(func)
+            for param in defaultParameters.parameters.values():
+                if ( param.default is not param.empty):
```

Comment:
i think you can write the nested if stmt in one line

Reply:
done